src/mamba/py_interface.cpp

```diff
@@ -75,7 +75,7 @@ PYBIND11_MODULE(mamba_api, m)
         .def("clear", &MRepo::clear);
 
     py::class_<MTransaction>(m, "Transaction")
-        .def(py::init<MSolver&, MultiPackageCache&, const std::string&>())
```

Comment:
this will be a problem for `boa`.

Reply:
yes I bet downstream projects will be impacted. the current signature is unrelevant regarding multiple caches